src/prepare_data/task_prepare_vaccination_data.py

```diff
@@ -122,7 +123,7 @@ def _clean_vaccination_data(df):
 
 
 def _plot_series(sr, title, label=None):
-    fig, ax = plt.subplots(figsize=(15, 5))
+    fig, ax = plt.subplots(figsize=PLOT_SIZE)
```

Comment:
You could also set in the `matplotlibrc` under `figure.figsize: 12, 8` for all newer plots.

Reply:
Not now but will do that later. Thanks for the pointer!